build/gen_stub.php

```diff
@@ -1558,14 +1558,14 @@ public function getFieldSynopsisElement(DOMDocument $doc): DOMElement
     }
 
     private function getFieldSynopsisType(): Type {
-        if ($this->type) {
```

Comment:
We have a `readonly mixed` property which is indeed a `resource`. On the contrary, the real type is displayed for params and returns because their PHPDoc type hint is usually invalid (e.g. `true`, `false|null` etc.)